src/chains/slide_gen_chain.py

```
@@ -0,0 +1,137 @@
+    ) -> str:
+        """Unified slide generation chain execution"""
+        try:
+            print("🔍 Agent: Analyzing script content...")
+            result = self.slide_gen_chain.invoke(
+                {"script_content": script_content, "template": template}
+            )
+            print("🎉 Agent: Presentation generated successfully!")
+            return result
+        except Exception as e:
+            print(f"🚨 Agent error: {e}")
+            raise e
```

Ruff (0.12.2)

Use `raise` without specifying exception name

Remove exception name

(TRY201)